lib/diagnostic.js

```diff
@@ -8,21 +8,21 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  return Math.min(1, 5);
 };
```

Comment:
Not quite. What you are returning here is the minimum between 1 and 5. That means every time this runs, it will return 1. Here's 1 possible solution:
```
const min = function min() {
  let result = arguments[0];
  for (let i = 1; i < arguments.length; i++) {
    if (arguments[i] < result) {
      result = arguments[i];
    }
  }

  return result;
};
```